packages/analytics/analytics-chart/src/components/chart-types/TimeSeriesChart.vue

```diff
@@ -176,7 +190,15 @@ const htmlLegendPlugin = {
   },
 }
 
-const plugins = computed(() => [htmlLegendPlugin, highlightPlugin, ...(props.type === 'timeseries_line' ? [verticalLinePlugin] : [])])
+const plugins = computed(() => [
+  htmlLegendPlugin,
+  highlightPlugin,
+  ...(props.zoom ? [dragSelectPlugin] : []),
+  ...(props.type === 'timeseries_line' ? [verticalLinePlugin] : []),
+])
+
+const remountLineKey = computed(() => plugins.value.map(p => p.id).join('-'))
+const remountBarKey = computed(() => plugins.value.map(p => p.id).join('-'))
```

Comment:
Had to make two distinct keys even though they're the same, because of this: 
<img width="385" alt="image" src="https://github.com/user-attachments/assets/5af29d23-b2ee-473f-8c18-653f5464f5e5">

Reply:
I think this is telling you that it's using the value of the key to manage the DOM: https://v3-migration.vuejs.org/breaking-changes/key-attribute

Maybe prefix the computed value with `'line'` or `'bar'`?

Reply:
done 👍